frontend/talentsearch/resources/js/components/skills/SkillsInDetail/SkillsInDetail.tsx

```diff
@@ -0,0 +1,172 @@
+      <h2 data-h2-font-size="b(h3)">
+        {intl.formatMessage({
+          defaultMessage: "3. Skills in detail",
+          description:
+            "Heading for skills in detail section of experience form.",
+        })}
+      </h2>
+      <p>
+        {intl.formatMessage({
+          defaultMessage:
+            "After adding some skills please explain how you used them, try answering one or more of these questions:",
+          description:
+            "Description for skills in detail section of experience form.",
+        })}
+      </p>
+      <ul
+        data-h2-margin="b(bottom, m)"
+        data-h2-font-color="b(darkgray)"
+        data-h2-font-style="b(italic)"
+      >
+        <li>
+          {intl.formatMessage({
+            defaultMessage:
+              "What did you accomplish, create or deliver using this skill?",
+            description:
+              "Question for user in skills in detail section of experience form.",
+          })}
+        </li>
+        <li>
+          {intl.formatMessage({
+            defaultMessage:
+              "What tasks or activities did you do that relate to this skill?",
+            description:
+              "Question for user in skills in detail section of experience form.",
+          })}
+        </li>
+        <li>
+          {intl.formatMessage({
+            defaultMessage:
+              "Were there any special techniques or approaches that you used?",
+            description:
+              "Question for user in skills in detail section of experience form.",
+          })}
+        </li>
+        <li>
+          {intl.formatMessage({
+            defaultMessage:
+              "How much responsibility did you have in this role?",
+            description:
+              "Question for user in skills in detail section of experience form.",
+          })}
+        </li>
+      </ul>
+      <div
+        data-h2-padding="b(all, m)"
+        data-h2-bg-color="b(lightgray)"
+        data-h2-radius="b(s)"
+      >
+        {skills.length > 0 ? (
+          skills.map(({ id, name, key }, index) => (
+            <React.Fragment key={key}>
+              <div
+                data-h2-display="b(flex)"
+                data-h2-justify-content="b(space-between)"
+              >
+                <p data-h2-font-weight="b(700)">
+                  {index + 1}. {name[locale]}
+                </p>
+                <Button
+                  color="secondary"
+                  mode="inline"
+                  data-h2-padding="b(top-bottom, s) b(right-left, none)"
+                  data-h2-font-style="b(underline)"
+                  data-h2-display="b(flex)"
+                  data-h2-align-items="b(center)"
+                  onClick={handleDelete}
+                >
+                  <TrashIcon style={{ width: "1rem" }} />
+                  <span
+                    data-h2-padding="b(left, xs)"
+                    data-h2-visibility="b(invisible) s(visible)"
```

Comment:
Haha, clever, I thought the text was missing but it was just collapsed.  Nice touch.